ch_7_Finetune/7_1_Finetune_Llama2-7B.md

```diff
@@ -0,0 +1,321 @@
+# 7.1 Finetuning Llama 2 (7B) using QLoRA
+
+To help you better understand the finetuning process, in this tutorial, we provide a practical guide leveraging BigDL-LLM to tune a large language model to a specific task.  [Llama-2-7b-hf](https://huggingface.co/meta-llama/Llama-2-7b-hf) is used as an example here to adapt the text generation implementation.
+
+## 7.1.1 Enable BigDL-LLM on Intel GPUs
+
+
+### 7.1.1.1 Install BigDL-LLM on Intel GPUs
+
+After following the steps in [Readme](./README.md#70-environment-setup)  to set up the environment, you can install BigDL-LLM in terminal with the command below:
+```bash
+pip install bigdl-llm[xpu] -f https://developer.intel.com/ipex-whl-stable-xpu
```

Comment:
Some dependencies are missed here. Please refer to [here](https://github.com/intel-analytics/BigDL/tree/main/python/llm/example/GPU/QLoRA-FineTuning#1-install)